src/query/api/v1/httpd/handler.go

```diff
@@ -199,7 +198,7 @@ func (h *Handler) registerHealthEndpoints() {
 
 // Endpoints useful for profiling the service
 func (h *Handler) registerProfileEndpoints() {
-	h.Router.HandleFunc(pprofURL, pprof.Profile)
+	h.Router.PathPrefix("/debug/pprof/").Handler(http.DefaultServeMux)
```

Comment:
Make this a `const` to match the pattern?

Reply:
Actually i kind of prefer this instead

Reply:
Yeah me too :) 

The const makes this harder to follow, not easier.